internal/service/ipam/api_dhcp_host_data_source_test.go

```diff
@@ -0,0 +1,100 @@
+package ipam_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/infobloxopen/bloxone-go-client/ipam"
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/acctest"
+)
+
+func TestAccDhcpHostDataSource_Filters(t *testing.T) {
+	dataSourceName := "data.bloxone_dhcp_hosts.test"
+	var v ipam.IpamsvcHost
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDhcpHostDataSourceConfigFilters("TF_TEST_HOST_01"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDhcpHostExists(context.Background(), dataSourceName, &v),
+					resource.TestCheckResourceAttr(dataSourceName, "results.#", "1"),
+					resource.TestCheckResourceAttr(dataSourceName, "results.0.name", "TF_TEST_HOST_01"),
+					resource.TestCheckResourceAttrSet(dataSourceName, "results.0.id"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccDhcpHostDataSource_TagFilters(t *testing.T) {
+	dataSourceName := "data.bloxone_dhcp_hosts.test_tag"
+	var v ipam.IpamsvcHost
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDhcpHostDataSourceConfigTagFilters("site1"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDhcpHostExists(context.Background(), dataSourceName, &v),
+					resource.TestCheckResourceAttr(dataSourceName, "results.#", "1"),
+					resource.TestCheckResourceAttr(dataSourceName, "results.0.name", "TF_TEST_HOST_01"),
+					resource.TestCheckResourceAttr(dataSourceName, "results.0.tags.location", "site1"),
```

Comment:
you should use a more specific tag, especially since all of the examples uses this same tag.